content/docs/dev/concepts/explore-tikv-features/backup-restore.md

```diff
@@ -10,12 +10,156 @@ menu:
+- `backup-data-size(after-compressed)`: The backup file size.
+
+#### Restore Raw Data
+
+To restore raw data to the cluster, execute the `tikv-br restore raw` command. To get help on this command, execute `tikv-br restore raw -h` or `tikv-br restore raw --help`.
+For example, restore the raw backup files in s3 `/backup-data/2022-09-16` to `TiKV` cluster.
+
+```
+export AWS_ACCESS_KEY_ID=&{AWS_KEY_ID};
+export AWS_SECRET_ACCESS_KEY=&{AWS_KEY};
+tikv-br restore raw \
+    --pd "${PDIP}:2379" \
+    --storage "s3://backup-data/2022-09-16/" \
+    --ratelimit 128 \
+    --log-file restoreraw.log
+```
+Explanations for some options in the above command are as follows:
+
+- `--ratelimit`: The maximum speed at which a restoration operation is performed (MiB/s) on each `TiKV` node.
+- `--log-file`: Writing the TiKV-BR log to the `restorefull.log` file.
+
+A progress bar is displayed in the terminal during the restoration. When the progress bar advances to 100%, the restoration is complete. The progress bar is displayed as follows:
+```
+tikv-br restore raw \
+    --pd "${PDIP}:2379" \
+    --storage "s3://backup-data/2022-09-16/" \
+    --ratelimit 128 \
+    --log-file restoreraw.log
+Restore Raw <---------/...............................................> 17.12%.
+```
+
+After restoration finish, the result message is displayed as follows:
+```
+[2022/09/20 18:02:12.540 +08:00] [INFO] [collector.go:67] ["Raw restore success summary"] [total-ranges=3] [ranges-succeed=3] [ranges-failed=0] [restore-files=3] [total-take=950.460846ms] [restore-data-size(after-compressed)=78.3MB] [total-kv=100000] [total-kv-size=108.7MB] [average-speed=114.4MB/s]
+```
+Explanations for the above message are as follows: 
+- `total-ranges`: Number of ranges that the whole backup task is split to. Equals to `ranges-succeed` + `ranges-failed`.
+- `ranges-succeed`: Number of succeeded ranges.
+- `ranges-failed`: Number of failed ranges.
+- `restore-files`: Number of restored files.
+- `total-take`: The restoration duration.
+- `total-kv`: Total restored kv count.
+- `total-kv-size`: Total restored kv size. Note that this is the original size before compression.
+- `average-speed`: The restoration speed, which approximately equals to `total-kv-size` / `total-take`.
+- `restore-data-size(after-compressed)`: The restoration file size.
+
+
+### Data Verification of Backup & Restore
+
+TiKV-BR can do checksum between TiKV cluster and backup files after backup or restoration finish with the config `--checksum=true`. Checksum is using the [checksum](https://github.com/tikv/client-go/blob/ffaaf7131a8df6ab4e858bf27e39cd7445cf7929/rawkv/rawkv.go#L584) interface in TiKV [client-go](https://github.com/tikv/client-go), which send checksum request to all TiKV regions to calculate the checksum of all **VALID** data. Then compare to the checksum value of backup files which is calculated during backup process.
```

Comment:
TODO: change link of `checksum` after we add a `checksum` introduction [here](https://deploy-preview-325--tikv.netlify.app/docs/dev/develop/rawkv/introduction/).